src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -26,17 +27,15 @@ export function MovieDetails({ movie }: MovieDetailsProps) {
   } = movie;
   return (
     <MovieDetailsContainer>
-      <ImageContainer>
-        <img alt={name} src={imageUrl} />
-      </ImageContainer>
+      <MovieImage alt={name} src={imageUrl} />
       <InfoContainer>
         <InfoHeader>
           {name} <Rating>{rating}</Rating>
         </InfoHeader>
         <Genre>{genreList.join(', ')}</Genre>
         <AdditionalInfo>
           <span> {releaseDate?.getFullYear()}</span>
-          <span>{duration}</span>
+          <span>{convertDuration(duration)}</span>
```

Comment:
Please fix test for 
`MovieDetails.test.tsx`
then.

But first we need to update `export const MOVIE_MOCK: Movie = {` to be consistent to the interface